### PR TITLE
docs(angular): Add troubleshooting for Angular 15 polyfills

### PR DESCRIPTION
**Pull request by wlee221**
#### Description of changes

Adds docs on how to polyfill `global` and `process` on Angular 15.

Angular 15 doesn't create `src/polyfills.ts` anymore (see "CLI improvements" section on [blog](https://blog.angular.io/angular-v15-is-now-available-df7be7f2f4c8)), so developers need to manually create that file now.

#### Description of how you validated changes

https://user-images.githubusercontent.com/43682783/223600470-a0a00509-8ee7-4678-a0a5-02886038c7fd.mp4

#### Checklist

- [x] PR description included
- [x] `yarn test` passes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by 0618 — APPROVED** [review]

File: docs/src/pages/[platform]/getting-started/troubleshooting/troubleshooting.angular.mdx [hunk 1]
```diff
@@ -9,6 +16,45 @@ Angular 6+ no longer include shims for 'global' or 'process'. Add the following
...
+```json{3}
+"polyfills": [
+  "zone.js",
+  "src/polyfills.ts"
+],
+```
+
+
+
+And finally, make sure to add `src/polyfills` to `files` in your `tsconfig`:
```
Comment (wlee221) [inline docs/src/pages/[platform]/getting-started/troubleshooting/troubleshooting.angular.mdx:30]:
```suggestion
```

And finally, make sure to add `src/polyfills` to `files` in your `tsconfig`:
```

Comment (wlee221) [inline docs/src/pages/[platform]/getting-started/troubleshooting/troubleshooting.angular.mdx:30]:
```suggestion

And finally, make sure to add `src/polyfills` to `files` in your `tsconfig`:
```

**Review by 0618 — APPROVED** [review]
LGTM

**Review by zchenwei — APPROVED** [review]
LGTM

**Pull request merged**